packages/event-cache/migrations/20190705152855-add-event-id.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('event', 'id', {
+      type: Sequelize.STRING
```

Comment:
Is index the primary key that replaces (blockNumber, logIndex) as primary key ?

BTW: since this table is empty in prod, as opposed to writing migrations to add columns,  it would be fine to start defining this entire table from scratch.

Reply:
not an index.  This was added because it's data returned from web3 for an event.  Since we want to re-assemble the event objects the same as web3 returns them, we want to include everything.

Reply:
Got it. Sounds good. Maybe add a comment in the model to indicate the role of the id column.